r/vignettes/developers/workflow.Rmd

```diff
@@ -4,6 +4,23 @@
 knitr::opts_chunk$set(error = TRUE, eval = FALSE)
 ```
 
+The Arrow R package uses several additional development tools:
+
+* [`lintr`](https://github.com/r-lib/lintr) for code analysis
+  - for the time being, the R package uses a custom version of lintr - `jonkeane/lintr@arrow-branch`
+* [`styler`](https://styler.r-lib.org) for code styling
+* [`pkgdown`](https://pkgdown.r-lib.org) for building the website
+* [`roxygen2`](https://roxygen2.r-lib.org) for documenting the package
+  - the R documentation uses the [`@examplesIf`](https://roxygen2.r-lib.org/articles/rd.html#functions) tag introduced in `roxygen2` version 7.1.2
+
+You can install all these additional dependencies by running:
+
+```{r, eval=FALSE}
```

Comment:
If it is just a code block, the eval option is not needed.

```suggestion
```r
```

Reply:
Good catch. Thanks